catalogs/catalog-jdbc-postgresql/src/test/java/com/datastrato/gravitino/catalog/postgresql/integration/test/CatalogPostgreSqlIT.java

```diff
@@ -853,7 +855,7 @@ void testUpdateColumnDefaultValue() {
   void testColumnDefaultValueConverter() {
     // test convert from MySQL to Gravitino
     String tableName = GravitinoITUtils.genRandomName("test_default_value");
-    String fullTableName = TEST_DB_NAME + "." + schemaName + "." + tableName;
+    String fullTableName = TEST_DB_NAME.toString() + "." + schemaName + "." + tableName;
```

Comment:
I guess `toString()` is unnecessary, right?

Reply:
For uppercase database name like `PG_TEST_ICEBERG_CATALOG_MULTIPLE_JDBC_LOAD`,
PostgreSQL will auto-convert this uppercase name to lowercase: e.g. `pg_test_iceberg_catalog_multiple_jdbc_load` , but enum value will be uppercase name, this 
make the tests failed because we can't find db name `PG_TEST_ICEBERG_CATALOG_MULTIPLE_JDBC_LOAD`.

So, we need to override `toString()` to return lowercase name.

Reply:
Oh, I see.

Reply:
Sorry, we don't need to add `toString()` at `TEST_DB_NAME`, because java String concatenation automatically invoke this method.

> String conversions are implemented through the method toString, defined by Object and inherited by all classes in Java.

I've already fixed it.

Reference:
- [String (Java Platform SE 8)](https://docs.oracle.com/javase/8/docs/api/java/lang/String.html)